### ISSUE TITLE
[BUG]  Broken link in documentation

### ISSUE THREAD

**Issue by MagicJF** [Jermolene/TiddlyWiki5#7763]
### Describe the bug

The examples section of this page  outputs a 404 error: 
https://tiddlywiki.com/static/function%2520Operator.html

Please repair the link because I cant figure out how to use this feature without examples.

### TiddlyWiki Configuration

- Version [e.g. v5.1.24]
- Saving mechanism [e.g. Node.js, TiddlyDesktop, TiddlyHost etc]
- Plugins installed [e.g. Freelinks, TiddlyMap]

**Comment by pmario** [comment]
You are right, there are no examples atm. You may go to the [Talk TiddlyWiki](https://talk.tiddlywiki.org/latest) discussion group and search for `\function`. You should find some useful posts there. 

If not, just ask ;)

**Comment by pmario** [comment]
Seems to be fixed with the current version
